lib/did_you_mean/test_helper.rb

```diff
@@ -1,7 +1,7 @@
 module DidYouMean
   module TestHelper
     def assert_suggestion(array, expected)
-      assert_equal [expected], array, "Expected #{array.inspect} to only include #{expected.inspect}"
+      assert_equal Array(expected), array, "Expected #{array.inspect} to only include #{expected.inspect}"
```

Comment:
Apologies for multiple changes in commit.  Did you want me to make these changes and re-send a pull request?